arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -0,0 +1,642 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-06-27-preview",
+    "title": "ContainerRegistryManagementClient"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "Operation"
+        ],
+        "description": "Checks whether the container registry name is available.",
+        "operationId": "Operation_CheckNameAvailability",

```

Comment:
Our RP explicitly put this under Operation since the path goes to Microsoft.ContainerRegistry/checkNameAvailability rather than, for example, Microsoft.ContainerRegistry/registries/{registryName}. I have seen different patterns. For example, storage account put checkNameAvailability under StorageAccount, while CDN put it under a special NameAvailability: https://github.com/Azure/azure-rest-api-specs/blob/master/arm-cdn/2016-10-02/swagger/cdn.json#L1594 Is this a required change or can we keep it as operation?